op-batcher/batcher/driver.go

```diff
@@ -124,6 +124,9 @@ type BatchSubmitter struct {
 	channelMgrMutex sync.Mutex // guards channelMgr and prevCurrentL1
 	channelMgr      *channelManager
 	prevCurrentL1   eth.L1BlockRef // cached CurrentL1 from the last syncStatus
+
+	blocks     map[common.Hash]*types.Block
```

Comment:
Can be removed.

Reply:
Removed in https://github.com/EspressoSystems/optimism-espresso-integration/pull/96/commits/99e32de72a7e322476c2183523d45f47cf0d014c.